src/mappings.py

```diff
@@ -1,7 +1,7 @@
 from typing import Type, Any
 
 from sqlalchemy import BIGINT, BOOLEAN, VARCHAR, DATE, TIMESTAMP
-from sqlalchemy.dialects.postgresql import BYTEA, DOUBLE_PRECISION
+from sqlalchemy.dialects.postgresql import BYTEA, DOUBLE_PRECISION, NUMERIC, ARRAY
```

Comment:
Are we using these new values yet?

Reply:
No, that's my mistake because I uncommented the usage of those types further down in the actual mapping dict. I'll remove the imports.